lc2fen/split_board.py

```diff
@@ -21,6 +21,12 @@ def split_board_image_trivial(
     individual square is saved into `out_dir` as `output_name` appended
     with its corresponding position on the chessboard.
 
+    For example, if `output_name` is `"square"` and `output_dir` is
+    `"Squares"`, then the "Squares" folder will be created in the
+    current directory and the image corresponding to the a8 square of
+    `board_image` will have the "Squares_0_0.jpg" filename and saved
```

Comment:
Wouldn't this be "square_0_0.jpg"? Also in the other example you added.

Reply:
You're right! I'll fix this blunder (silently in the next commit)

Reply:
Done